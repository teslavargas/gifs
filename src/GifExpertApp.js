import React, { useState } from 'react'
import { AddCategory } from './components/AddCategory';
import { GifGrid } from './components/GifGrid';

export const GifExpertApp = () => {

    const [categories, setCategories] = useState(['Dua Lipa']); 

    return (
        <>
            <h2 className="animate__animated animate__bounceInDown">GifExpertApp</h2>
            <AddCategory setCategories={setCategories}/> 
            <hr/>
           

            <ol>
                {
                    categories.map(category => ( 
                        <GifGrid key={category} category={category}/>
                    ))
                }

            </ol>
        </>
    )
}
