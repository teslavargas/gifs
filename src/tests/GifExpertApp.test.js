import React from 'react'; 
import { shallow } from 'enzyme'; 
import GifExpertApp  from '../GifExpertApp';

describe('Pruebas en <GifExpertApp />', () => {
    test('Debe mostrarse correctamente', () => {
        const wrapper = shallow(<GifExpertApp/>); 
        expect(wrapper).toMatchSnapshot();

    })
    
})
