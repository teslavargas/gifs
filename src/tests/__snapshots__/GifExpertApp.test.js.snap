// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> Debe mostrarse correctamente 1`] = `
<Fragment>
  <h2
    className="animate__animated animate__bounceInDown"
  >
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category={Array []}
      key=""
    />
  </ol>
</Fragment>
`;
